web-ui/src/main/resources/catalog/js/admin/DashboardStatusController.js

```diff
@@ -168,11 +168,7 @@
       };
 
       $scope.downloadLog = function() {
-        $http.get('../api/site/logging/activity', null, {
-          headers: {
-            'Content-Type': 'application/zip'
-          }
-        });
+        window.location = '../api/site/logging/activity/zip';
```

Comment:
Wonder whether there is a place where this make a difference (involving troubles).